### DISCUSSION TITLE
AVD 4.3 install issue

### DISCUSSION THREAD

**Discussion by suspectzed** [aristanetworks/avd#3131] [General]
I might be missing something.
Following:
https://avd.arista.com/4.3/docs/installation/collection-installation.html

mkdir /home/aedwards/arista
cd /home/aedwards/arista
ansible-galaxy collection install arista.avd (no issue)
pip3 install -r /home/aedwards/.ansible/collections/ansible_collections/arista/avd/requirements.txt (no issue)
ansible-playbook arista.avd.install_examples
ERROR! the playbook: arista.avd.install_examples could not be found

It's there though:
ls /home/aedwards/.ansible/collections/ansible_collections/arista/avd/playbooks | grep install_examples
install_examples.yml

and I can get it directly (not through the collection):
ansible-playbook /home/aedwards/.ansible/collections/ansible_collections/arista/avd/playbooks/install_examples.yml

splunk@containerlab:~/arista$ ansible --version
ansible 2.10.8
  config file = None
  configured module search path = ['/home/splunk/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]

Anyone have any ideas?

**Comment by suspectzed** [comment]
RTFM!  Apparently Arista AVD doesn't support ansible 2.10.8

**Reply by ClausHolbechArista** [reply]
Correct, we currently support `ansible-core from 2.12.6 to 2.15.x excluding 2.13.0`.

**Comment by gmuloc** [comment]
Closing as auto answered!
